domain/port/service/watcher.go

```diff
@@ -101,34 +104,61 @@ func (s *WatchableService) WatchOpenedPortsForApplication(ctx context.Context, a
 func (s *WatchableService) endpointToMachineMapper(
```

Comment:
There's a type in the comment to this method too, though not part of the change set.